index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family='Poppins':wght@400;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>My mobile portfolio</title>
+  </head>
+  <body>
+    <main>
+      <nav class="portfolio-nav">
+        <h2 class="portfolio-nav--logo">My logo</h2>
```

Comment:
Kindly put your logo inside the `a` tag because it will increase the SEO for your website. This is essentially for three reasons: convention, semantics, and user experience(UX).
[Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.